management/hook_secrets_test.go

```diff
@@ -0,0 +1,69 @@
+package management
+
+import (
+	"gopkg.in/auth0.v4/internal/testing/expect"
+	"testing"
+
+	"gopkg.in/auth0.v4"
+)
+
+func TestHookSecrets(t *testing.T) {
+
+	r := &map[string]string{
+		"SECRET1": "value1",
+		"SECRET2": "value2",
+	}
+
+	hook := &Hook{
+		Name:      auth0.String("test-hook-secrets"),
+		Script:    auth0.String("function (user, context, callback) { callback(null, { user }); }"),
+		TriggerID: auth0.String("pre-user-registration"),
+		Enabled:   auth0.Bool(false),
+	}
+
+	err := m.Hook.Create(hook)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Cleanup(func() {
```

Comment:
I was not aware of this function! Thanks! TIL.

Reply:
I wasn't aware of it until my IDE told me it existed :)